mergify_oauth/README.md

```diff
@@ -0,0 +1,69 @@
+# Mergify
+
+## Overview
+
+This integration monitors merge queue length for each configured repository in
+[Mergify][1] and tracks Mergify's global availability. By sending metrics to your
+Datadog account, you can set up monitors for anomaly alerts and analyze merge
+queue performance. You can maintain awareness of Mergify service availability
+and optimize your development workflow using this Datadog integration.
+
+## Setup
+
+1. Go to the [Mergify dashboard][3], log in, and navigate to **Integrations**.
+2. Click the **Connect** button in the **Datadog** tile and you will be redirected to the Datadog authorization page.
```

Comment:
We recommend instructing the user to start the flow from the Datadog side. They should find the Mergify integration in Datadog and click 'connect accounts' from there.